Tests/LibWeb/Layout/expected/block-and-inline/inline-block-leading-and-trailing-metrics.txt

```diff
@@ -18,13 +18,13 @@ Viewport <#document> at (0,0) content-size 800x600 children: not-inline
       BlockContainer <(anonymous)> at (8,28) content-size 784x0 children: inline
         TextNode <#text>
       BlockContainer <div#b> at (9,29) content-size 782x18 children: inline
-        frag 0 from BlockContainer start: 0, length: 0, rect: [70.296875,29 88.453125x18] baseline: 13.796875
+        frag 0 from BlockContainer start: 0, length: 0, rect: [50.296875,29 88.453125x18] baseline: 13.796875
```

Comment:
Please update test results in the same commit that modifies them. (So, I'm assuming this and most of these others should go in the previous commit.)

Adding *new* tests in a separate commit is fine.